runners/samza/src/main/java/org/apache/beam/runners/samza/translation/ConfigBuilder.java

```diff
@@ -255,9 +230,40 @@ private static void validateYarnRun(Map<String, String> config) {
       configBuilder.put("job.host-affinity.enabled", "true");
     }
 
+    switch (options.getSamzaExecutionEnvironment()) {
```

Comment:
not sure whether we have logged the entire pipeline option somewhere. Otherwise it would be good to have a log entry here about which execution environment we're running

Reply:
Makes sense. Added more logs.